### PR TITLE
Add Marathon deploy template

### PR DESCRIPTION
**Pull request by declantraynor**

### CODE REVIEW

File: deploy.json.tmpl [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "id": "${CS_MARATHON_APP_ID}",
+  "cmd": null,
+  "cpus": 0.2,
+  "mem": 512,
```
Comment (paddycarey) [inline deploy.json.tmpl]: gddo is stateless, right? could we drop this memory figure way down and save some cluster resources? say 128?

Comment (declantraynor) [inline deploy.json.tmpl]: @paddycarey good spot, let's do it.

**Comment by paddycarey** [comment]
LGTM :+1:

**Comment by tyndyll** [comment]
:+1:

**Pull request merged**